beacon-chain/rpc/validator/aggregator.go

```diff
@@ -68,7 +68,7 @@ func (vs *Server) SubmitAggregateSelectionProof(ctx context.Context, req *ethpb.
 	if len(aggregatedAtts) == 0 {
 		aggregatedAtts = vs.AttPool.UnaggregatedAttestationsBySlotIndex(req.Slot, req.CommitteeIndex)
 		if len(aggregatedAtts) == 0 {
-			return nil, status.Errorf(codes.Internal, "Could not find attestation for slot and committee in pool")
+			return nil, status.Errorf(codes.Unavailable, "Could not find attestation for slot and committee in pool")
```

Comment:
This isn't appropriate. Unavailable signals to the client that it should retry later. It's also the default when there is no beacon node serving a request.

What about NotFound as i recommended in the issue description https://github.com/prysmaticlabs/prysm/issues/7859?

Reply:
ops, i meant to use `NotFound` here... Fixed. Thanks!